src/backend/metal/src/command.rs

```diff
@@ -1336,9 +1336,7 @@ impl RawCommandQueue<Backend> for CommandQueue {
                     trace!("\timmediate {:?} with {} passes", token, num_passes);
                     self.retained_buffers.extend(retained_buffers.drain(..));
                     self.retained_textures.extend(retained_textures.drain(..));
-                    if num_passes != 0 {
-                        cmd_buffer.commit();
-                    }
+                    cmd_buffer.commit();
```

Comment:
Why did we have to back out the `num_passes` check? It seems reasonable unless need to attach completion handlers, but maybe we could handle that some other way

Reply:
oh, that looks like a mistake on my part